site/public/js/drag-and-drop.js

```diff
@@ -301,40 +301,47 @@ function removeLabel(filename, part){
 
 function addLabel(filename, filesize, part, previous){
     // create element
-    var tmp = document.createElement('label');
-    tmp.setAttribute("class", "mylabel");
-    tmp.setAttribute("fname", filename);
-    tmp.innerHTML =  filename + " " + filesize + "kb <i role='text' aria-label='Press enter to remove file " + filename + "' tabindex='0' class='fas fa-trash custom-focus'></i><br />";
+    var uploadRowElement = document.createElement('tr');
+    uploadRowElement.setAttribute("class", "label");
+    uploadRowElement.setAttribute("fname", filename);
 
+    var fileDataElement = document.createElement('td');
+    var fileTrashElement = document.createElement('td');
+
+    fileDataElement.innerHTML= filename;
+    fileTrashElement.innerHTML= filesize + "KB  <i role='text' aria-label='Press enter to remove file " + filename + "' tabindex='0' class='fas fa-trash custom-focus'></i>";
+
+    uploadRowElement.appendChild(fileDataElement);
+    uploadRowElement.appendChild(fileTrashElement);
     // styling
-    tmp.children[0].onmouseover = function(e){
+    uploadRowElement.children[1].onmouseover = function(e){
         e.stopPropagation();
         this.style.color = "#FF3933";
     };
-    tmp.children[0].onmouseout = function(e){
+    uploadRowElement.children[1].onmouseout = function(e){
         e.stopPropagation();
         this.style.color = "black";
     };
-    // remove file and label on click
-    tmp.children[0].onclick = function(e){
+    // remove file and label-row in table on click event
+    uploadRowElement.children[1].onclick = function(e){
         e.stopPropagation();
         this.parentNode.parentNode.removeChild(this.parentNode);
         deleteSingleFile(filename, part, previous);
     };
 
     // FOR VPAT if trash can has focus and key is pressed it will delete item
-    tmp.children[0].onkeypress = function(e){
+    uploadRowElement.children[1].onkeypress = function(e){
```

Comment:
can you simplify all of these `uploadRowElement.children[1]` to just be `fileTrashElement`?